### PR TITLE
Add streamlit inference demo page

### PR DESCRIPTION
**Pull request by AppleHolic**
image
Hi! 

I'm very impressed with your research result, and also I want to test my samples as easily as possible.

So, I made a simple web-based demo using [streamlit](https://github.com/streamlit/streamlit).

### CODE REVIEW

**Comment by haoheliu** [comment]
Great feature! Thanks for your work! I'll test it now and get back to you soon.

**Comment by AppleHolic** [comment]
I missed the code of streamlit's location should be part of a package for deploying to pypi. 
I will push more commits to apply it

**Comment by AppleHolic** [comment]
I just move the streamlit code to test directory and update README.md!

File: test/streamlit.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+if w:
+    st.write('Inference : ')
+    
+    # choose options
+    mode = st.radio('Voice fixer mode (0: rm high frequency, 1: none, 2: train fixer)', [0, 1, 2])
+    if torch.cuda.is_available():
+        is_cuda = st.radio('Turn on GPU', [True, False])
+        if is_cuda != list(voice_fixer._model.parameters())[0].is_cuda:
+            device = 'cuda' if is_cuda else 'cpu'
+            voice_fixer._model = voice_fixer._model.to(device)
```
Comment (haoheliu) [inline test/streamlit.py:39]: Maybe here we need another else branch? I test it on my MacBook without GPU and it shows is_cuda is not defined in line 47.

```python3
    if torch.cuda.is_available():
        is_cuda = st.radio('Turn on GPU', [True, False])
        if is_cuda != list(voice_fixer._model.parameters())[0].is_cuda:
            device = 'cuda' if is_cuda else 'cpu'
            voice_fixer._model = voice_fixer._model.to(device)
    else:
        is_cuda = False
```

Comment (haoheliu) [reply test/streamlit.py:39]: OK, I'll merge it first and solve it by myself. Thanks for your work!

Comment (AppleHolic) [reply test/streamlit.py:39]: Ah.. Because my laptop has a GPU, I cannot found that point. Thanks!

**Pull request merged**